.github/workflows/run-tests.yml

```
@@ -25,7 +25,7 @@ jobs:
       - name: Setup
         uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c
         with:
-          node-version: 18.17.1
+          node-version: 18.20.4
```

I think we can actually change this to pick up the .nvmrc file instead ([we do this in the shared CI workflows](https://github.com/alphagov/pay-ci/blob/master/.github/workflows/_run-node-cypress-tests.yml)):

```suggestion
          node-version-file: ".nvmrc"
```